### ISSUE TITLE
featureCounts should be able to take a collection and output a single file: a counts matrix

### ISSUE THREAD

**Issue by alexlenail** [galaxyproject/tools-iuc#1740]
At present, featureCounts, when passed a collection, outputs a collection of vectors. E.g. it does
```
featureCounts 1.bam
featureCounts 2.bam
...
featureCounts n.bam
```
However, the featureCounts binary provides the ability to pass wildcards, creating counts matrices
```
featurecounts *.bam
```
It would be nice if, passed a collection, the featureCounts tool provided this latter option. 

(thanks to @jennaj for help)

**Comment by bernt-matthias** [comment]
My argument against it would be that in the current way parallelism is used and the created collection can easily transformed into a single table.

**Comment by bgruening** [comment]
@bernt-matthias maybe we can add this hint into the help section of the tool and explain how to proceed if the user needs a matrix?

**Comment by bernt-matthias** [comment]
Good idea: see here: https://galaxyproject.github.io/training-material/topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html#create-count-matrix

**Comment by bernt-matthias** [comment]
Can do.

Just thinking about an alternative : use read groups set in them Sam file. Any idea how this info could be added?

**Comment by bernt-matthias** [comment]
Some aligners could do this apparently. But either we would need to to implement the use of the element identifier or add the option if its missing (e.g. HISAT2 wrapper). 

Adding it after mapping, e.g. with samtools or picard, seems unfavorable since this just doubles hard disk memory consumption.

**Comment by SaimMomin12** [comment]
@bernt-matthias Can we close this?

**Comment by bernt-matthias** [comment]
Maybe extending docs would be a good idea https://github.com/galaxyproject/tools-iuc/issues/1740#issuecomment-526683108

**Comment by SaimMomin12** [comment]
Can we transfer this issue to the GTN repo?

**Comment by bernt-matthias** [comment]
I would suggest to update the tool help: https://github.com/galaxyproject/tools-iuc/blob/0e10347a5ab63b64d098829280fa44674573149e/tools/featurecounts/featurecounts.xml#L460

Currently the help mentions alignments but it only takes a single alignment .. and I guess it's confusing to the user since Galaxy allows them to provide alignments which are then processed independently.
